virtual/pandoc/pandoc-0.ebuild

```
@@ -0,0 +1,15 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Virtual for pandoc"
+
+LICENSE=""
+SLOT="0"
+KEYWORDS="amd64 ~arm64 ~riscv ~x86"
```

The issue is that `virtual/pandoc` has `KEYWORDS="amd64 ~arm64 ~riscv ~x86"` while `app-text/pandoc-cli` only has `KEYWORDS="~amd64"`.

You cannot have `KEYWORDS` on a package that are not present in the `KEYWORDS` of its dependencies.

If you have success building `app-text/pandoc-cli` on `arm64`, `riscv`, or `x86`, then I would have no problem keywording `app-text/pandoc-cli` and its dependencies for those architectures. Otherwise, just change it to:

```suggestion
KEYWORDS="~amd64"
```

---

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Ah, thank you; right, got it.  I don't have access to a 32-bit x86, ARM64, or RISC-V machine to test building `app-text/pandoc-cli` on, so I'll make the second change.